README.md

```diff
@@ -6,13 +6,16 @@
 [![License](https://img.shields.io/cocoapods/l/JWTDecode.svg?style=flat-square)](https://cocoadocs.org/docsets/JWTDecode)
 [![Platform](https://img.shields.io/cocoapods/p/JWTDecode.svg?style=flat-square)](https://cocoadocs.org/docsets/JWTDecode)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat-square)](https://github.com/Carthage/Carthage)
-![Swift 5.1](https://img.shields.io/badge/Swift-5.1-orange.svg?style=flat-square)
+![Swift 5.3](https://img.shields.io/badge/Swift-5.3-orange.svg?style=flat-square)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2FJWTDecode.swift.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2FJWTDecode.swift?ref=badge_shield)
 
 This library will help you check [JWT](https://jwt.io/) payload
 
 > This library doesn't validate the token, any well formed JWT can be decoded from Base64Url.
 
+## Important Notices
+- [The latest majors of Quick and Nimble](https://github.com/auth0/JWTDecode.swift/pull/114) require that developers using Carthage **must** use Xcode 11.4+ only, starting from JWTDecode.swift [2.5.0](https://github.com/auth0/JWTDecode.swift/releases/tag/2.5.0).
```

Comment:
I would update requirements to match 11.4+, give we're not testing on anything less now, can't say for sure that it works.

Reply:
I removed the notice and updated the Xcode requirements to 11.4+

Reply:
I'm pointing to this PR in order to use JWTDecode.swift in a Xcode 12 project, is going to be merged 🙂?